src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.util.Scanner;
+
+import commands.Command;
+import tasks.*;
+import exceptions.*;
+
+public class Duke {
+
+    private Ui ui;
+    private TaskList taskList;
+    private Storage store;
```

Comment:
Would `store` be a bit vague since `store` might have multiple meanings? (textbook page 62) 

Feel free to ignore !